tools/docker/BUILD

```diff
@@ -15,10 +15,39 @@
 ################################################################################
 #
 load("@io_bazel_rules_docker//container:image.bzl", "container_image")
+load(
+    "@io_bazel_rules_docker//container:container.bzl",
+    "container_push",
+)
 
 container_image(
-    name = "envoy",
+    name = "envoy_distroless",
     base = "@distroless_cc//image",
     tags = ["manual"],
     tars = ["//src/envoy:envoy_tar"],
 )
+
+container_image(
+    name = "envoy_ubuntu",
+    base = "@bionic//image",
+    tags = ["manual"],
+    tars = ["//src/envoy:envoy_tar"],
+)
+
+container_push(
+    name = "push_envoy_distroless",
+    format = "Docker",
+    image = ":envoy_distroless",
+    registry = "gcr.io",
+    repository = "{DOCKER_REPOSITORY}",
+    tag = "{BUILD_CONFIG}-{BUILD_SCM_REVISION}",
```

Comment:
Nit: don't we want this to be "dockerless-{BUILD_CONFIG}-{BUILD_SCM_REVISION}"?

Reply:
distroless seems to be the future default, so I thought I'd save on naming.

Reply:
K